ui/apps/platform/src/Containers/ComplianceEnhanced/Schedules/components/ReportJobs.tsx

```diff
@@ -1,7 +1,78 @@
 import React from 'react';
+import { Card, CardBody, Divider } from '@patternfly/react-core';
 
-function ReportJobs() {
-    return <div />;
+import {
+    ComplianceScanConfigurationStatus,
+    ComplianceScanSnapshot,
+} from 'services/ComplianceScanConfigurationService';
+import JobDetails from 'Containers/Vulnerabilities/VulnerablityReporting/ViewVulnReport/JobDetails';
+import ReportJobsTable from 'Components/ReportJobsTable';
+import ConfigDetails from './ConfigDetails';
+
+type ReportJobsProps = {
+    scanConfig: ComplianceScanConfigurationStatus | undefined;
+};
+
+function createMockData(scanConfig: ComplianceScanConfigurationStatus) {
+    const snapshots: ComplianceScanSnapshot[] = [
+        {
+            reportJobId: 'ab1c03ae-9707-43d1-932d-f948afb67b53',
+            reportStatus: {
+                completedAt: '2024-08-27T00:01:40.569402380Z',
+                errorMsg:
+                    "Error sending email notifications:  error: Error sending email for notifier 'fc99e179-57c1-4ba2-8e59-45dbf184c78c': Connection failed",
+                reportNotificationMethod: 'EMAIL',
+                reportRequestType: 'SCHEDULED',
+                runState: 'FAILURE',
+            },
+            user: {
+                id: 'sso:3e30efee-45f0-49d3-aec1-2861fcb3faf6:c02da449-f1c9-4302-afc7-3cbf450f2e0c',
+                name: 'Test User',
+            },
+            isDownloadAvailable: false,
+            scanConfig,
+        },
+    ];
+    return snapshots;
+}
+
+function getJobId(snapshot: ComplianceScanSnapshot) {
+    return snapshot.scanConfig.id;
+}
+
+function getConfigName(snapshot: ComplianceScanSnapshot) {
+    return snapshot.scanConfig.scanName;
+}
+
+function ReportJobs({ scanConfig }: ReportJobsProps) {
```

Comment:
Does `scanConfig` as prop imply that same (current) configuration for all report jobs?

Reply:
Yeah, I was trying to make it easier to make the mock data using the actual real config data. This is just for mocking data for now. This will be replaced ofc.